sfja/src/components/Upload/PreviewBlankForm.js

```diff
@@ -44,7 +47,7 @@ class PreviewBlankForm extends React.Component {
           </Button>
           <br/>
           <TextField onChange={(e) => {
-            this.setState({value: e.target.newName});
+            this.setState({value: e.target.value});
```

Comment:
also @trish234 this was your bug -- onChange's new values go to `e.target.value`! I fixed this and the back button now works.